src/main/resources/Service.csv

```diff
@@ -0,0 +1,2 @@
+service_code,ccd_service_name,ccd_jurisdiction_name
```

Comment:
is this file required in the src/main or test folder?

Reply:
right. removed